### PR TITLE
Cca 40 search for requirements txt

### PR DESCRIPTION
**Pull request by ola567**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -15,10 +16,15 @@
 OS_image="ubuntu"
 OS_image_version="latest"
 message = "testing message 123"
+path = 'C:\\Users\\aleks\\Desktop\\ContainerStudio\\outputs'
```
Comment (esesyn01) [inline main.py:19]: We're running this project on Linux, not Windows

File: requirements_searching.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                matching_files.append(relative_path)
+    return matching_files
+
+
+def _get_file_names(files):
+    file_names = [os.path.split(file)[-1] for file in files]
+    return file_names
+
+
+def _convert_paths_to_linux(paths):
```
Comment (esesyn01) [inline requirements_searching.py:20]: Again, we're running this project on Linux, so we don't need a function like this

File: requirements_searching.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    return [path.replace("\\", "/") for path in paths]
+
+
+def use_requirements(path):
+    pattern = re.compile(r".*requirements.*", re.IGNORECASE)
```
Comment (esesyn01) [inline requirements_searching.py:25]: Case in filenames matters on Linux, it should be avoided

File: main.py (hunk 1 above)
Comment (ola567) [reply main.py:19]: I know, but it works the same in Windows and Linux. Path doesn't matter.

File: requirements_searching.py (hunk 2 above)
Comment (ola567) [reply requirements_searching.py:20]: I can remove it.

File: requirements_searching.py (hunk 3 above)
Comment (ola567) [reply requirements_searching.py:25]: Sure, I will delete this.

**Pull request merged**